lib/instance_agent/plugins/codedeploy/command_executor.rb

```diff
@@ -473,6 +473,24 @@ def app_spec_path
           'appspec.yml'
         end
 
+        # Checks for existence the possible extensions of the app_spec_path (.yml and .yaml)
+        private
+        def app_spec_real_path(deployment_spec)
+          app_spec_yaml_location = File.join(archive_root_dir(deployment_spec), "appspec.yaml")
+          app_spec_yml_location = File.join(archive_root_dir(deployment_spec), "appspec.yml")
+          app_spec_default_location = File.join(archive_root_dir(deployment_spec), deployment_spec.app_spec_path)
```

Comment:
I wouldn't call app_spec_default_location "default", it's more of an override, right?
The other two could include "default" in their names.

Reply:
Makes sense, indeed this is the only one! I changed it to reflect what you proposed.